tests/etl/test_orchestrator.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+import sys
+import unittest
+from pathlib import Path
+
+
+class OrchestratorTests(unittest.TestCase):
+    def test_examples(self):
```

Comment:
How does this work? Will it the test runner understand that the sub-processes are unit tests? and will it propagate individual test results back to the top level process?

Reply:
I love the fact that this test enforces us to keep the example code alive and up to date

Reply:
This simply validates that the examples can be executed. Check=True checks that the process exits normally without exception (no error code).